README.md

```diff
@@ -1,25 +1,18 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-To Do List
-This project is to create a daily list of things to do. It involes being able to add items to o be done, to remove what has been done and finally to edit an item in case the need arises.
-
-Use webpack to bundle JavaScript.
-Learn how to use proper ES6 syntax.
-Use ES6 modules to write modular JavaScript.
 
+> To Do List. The To do list app is a project which allows you to write down a task that you want to do (you schedule a task) you can mark them as completed or edit them and finally delete or clear all the completed tasks which are marked as completed. Built With , HTML , CSS, JAVASCRIPT, ES6 MODULES , WEBPACK, LINTER
 
```

Comment:
* Kindly add setup instructions to your readme file to make your readme more descriptive as this would help visitors easily setup your app 👍